src/components/converter/Form.js

```diff
@@ -18,9 +21,8 @@ import Anchor from '../Anchor'
 import question from './assets/question.svg'
 
 const large = css => breakpoint('large', css)
-
-const ANJ_BY_ANT = 100
-const ANJ_MIN_REQUIRED = bigNum(10)
+const options = ['ANT', 'DAI', 'ETH', 'USDC']
+const ANJ_MIN_REQUIRED = bigNum(0)
```

Comment:
😅Changing the `bigNum(10)` to `bigNum(0)` means this value is quite a lot different than expected